docs/src/locators.md

```diff
@@ -806,14 +806,14 @@ await Expect(page.Locator("x-details")).ToContainTextAsync("Details");
 Consider the following DOM structure where we want to click on the buy button of the second product card. We have a few options in order to filter the locators to get the right one.
 
 ```html card
-<div role="listitem">
+<article>
```

Comment:
I think the list of products in a show is a list, not an article.

Reply:
The docs also said that product cards should be articles. They can be a list of articles for sure but the card itself an article cause that card can be placed on any page. It’s independent of the other cards. It can be in a list but the card is an independent item.